compiler/src/iree/compiler/GlobalOptimization/Passes.cpp

```diff
@@ -24,6 +24,11 @@ static llvm::cl::opt<bool> clEnableQuantizedMatmulReassociation(
     llvm::cl::desc(
         "Enables reassociation of quantized matmul ops (experimental)."),
     llvm::cl::init(false));
+static llvm::cl::opt<bool> clEnableFuseHorizontalMatmul(
+    "iree-global-opt-enable-fuse-horizontal-matmul",
```

Comment:
Maybe a more restricted naming would help. Maybe call it silu-horizontal-matmul-fusion.